### PR TITLE
LPS-170432 Extract config into a new method

### PR DESCRIPTION
**Pull request by ealonso**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by sandrodw3** [comment]
Hey @DiegoHu97! Very nice 😄 I'm leaving some little comments:
- As I told you in Slack, it's currently throwing a null pointer, when you can fix it, I'll test functionality
- I think we could also destructure `config` (as you did with `props`) so we don't have to put `config.whatever` everywhere
- This is personal, but I usually prefer passing all props to the components instead of wrapping them in `whateverComponentData`. I only do the wrapping when I specifically need it for some reason. I think the other way it's more clear and easier to read, what do you think?
- In case you leave it with the `data` pattern, we should rename `PagesAdministrationData` to `pagesAdministrationData`, as capital initial letters are usually reserved for components and not for variables

Please reach me if you have any doubt 😄

File: modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/js/PagesAdministrationLink.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+import ClayLink from '@clayui/link';
+import React from 'react';
+
+export default function PagesAdministrationLink({
+	administrationPortletURL,
+	hasAdministrationPortletPermission,
+}) {
+	return (
+		<>
```
Comment (sandrodw3) [inline modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/js/PagesAdministrationLink.js:23]: We don't need the `<>` here, as we are returning only one element 🙂

**Comment by DiegoHu97** [comment]
Just started reviewing :)

**Comment by DiegoHu97** [comment]
see https://github.com/liferay-echo/liferay-portal/pull/10654

**Pull request closed without merging**